core/src/test/java/org/opensearch/sql/expression/datetime/ConvertTZTest.java

```diff
@@ -134,37 +140,46 @@ public void invalidInput1() {
         DSL.literal("test"),
         DSL.literal("+00:00"),
         DSL.literal("+00:00"));
-    assertEquals(DATETIME, expr.type());
+    assertEquals(TIMESTAMP, expr.type());
     assertEquals(nullValue(), expr.valueOf());
   }
 
   @Test
   public void invalidDateFeb30() {
-    FunctionExpression expr = DSL.convert_tz(DSL.datetime(
-            DSL.literal("2021-02-30 10:00:00")),
-        DSL.literal("+00:00"),
-        DSL.literal("+00:00"));
-    assertEquals(DATETIME, expr.type());
-    assertEquals(nullValue(), expr.valueOf());
+    assertThrows(SemanticCheckException.class,
+        ()-> {
+          FunctionExpression expr = DSL.convert_tz(DSL.timestamp(
+                  DSL.literal("2021-02-30 10:00:00")),
+              DSL.literal("+00:00"),
+              DSL.literal("+00:00"));
+          assertEquals(TIMESTAMP, expr.type());
+          assertEquals(nullValue(), expr.valueOf());
+    });
   }
 
   @Test
   public void invalidDateApril31() {
-    FunctionExpression expr = DSL.convert_tz(DSL.datetime(
-            DSL.literal("2021-04-31 10:00:00")),
-        DSL.literal("+00:00"),
-        DSL.literal("+00:00"));
-    assertEquals(DATETIME, expr.type());
-    assertEquals(nullValue(), expr.valueOf());
+    assertThrows(SemanticCheckException.class,
+            ()-> {
```

Comment:
and here

Reply:
updated in https://github.com/Bit-Quill/opensearch-project-sql/commit/f6edc6be62abbe3a3974665e8d51c515584234e0